scipy/interpolate/meson.build

```diff
@@ -111,6 +111,15 @@ py3.extension_module('_ppoly',
   subdir: 'scipy/interpolate'
 )
 
+py3.extension_module('_rgi_cython',
+  linalg_cython_gen.process('_rgi_cython.pyx'),
```

Comment:
I'm not sure if this line is needed - it was copied from `_ppoly` but we are not using any linear algebra functions

Reply:
Yeah, likely not needed --- ppoly does call `dgeev` to diagonalize the companion matrix in the `roots` method. But this can be cleaned up later I think.